docs/tutorials/blinky/pinetime.rst

```diff
@@ -0,0 +1,186 @@
+Blinky, your "Hello World!", on a PineTime smartwatch
+-------------------------------------------------------
+
+This tutorial shows you how to create, build, and run the Blinky
+application on a PineTime smartwatch.
+
+.. contents::
+  :local:
+  :depth: 2
+
+Prerequisites
+~~~~~~~~~~~~~
+
+-  Meet the prerequisites listed in :doc:`Project Blinky <blinky>`.
+-  Have a `PineTime Dev Kit <https://store.pine64.org/?product=pinetime-dev-kit>`__
+-  Have a `ST-LINK programmer <https://www.st.com/en/development-tools/st-link-v2.html>`__ or a cheap clone
+-  Install a patched version of OpenOCD 0.10.0 described in :doc:`Install OpenOCD <../../get_started/native_install/cross_tools>`.
+
+If you have not removed the flash protection yet, you should to that first. 
+See `this article <https://github.com/lupyuen/visual-embedded-rust/blob/master/README.md>`__ 
+for instructions.
```

Comment:
I think this page would be more suitable here, since it explains all the possible methods and it's the manufacturer's wiki. https://wiki.pine64.org/index.php/Reprogramming_the_PineTime

Reply:
You are right. That is a better page. I did actually use that own to click through to that other article ;-)